src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVKmerizerWithLowComplexityFilter.java

```diff
@@ -0,0 +1,90 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import java.util.Arrays;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+import static org.broadinstitute.hellbender.tools.spark.sv.SVKmer.Base;
+
+/** iterator over kmers with a specified minimum base-wise shannon entropy */
+public class SVKmerizerWithLowComplexityFilter extends SVKmerizer {
+    private final double minEntropy;
+    private final int[] baseCounts;
+    private static double[] entropies;
+
+    public SVKmerizerWithLowComplexityFilter( final byte[] seq, final int kSize, final double minEntropy ) {
+        this(new ASCIICharSequence(seq), kSize, minEntropy);
+    }
+
+    public SVKmerizerWithLowComplexityFilter( final CharSequence seq, final int kSize, final double minEntropy ) {
+        super(kSize, seq);
+        this.minEntropy = minEntropy;
+        baseCounts = new int[Base.values().length];
+        baseCounts[(int)Base.A.value] = kSize;
+        this.nextKmer = nextKmer(new SVKmer(kSize), 0);
+    }
+
+    public static Stream<SVKmer> stream( final CharSequence seq, final int kSize, final double minEntropy ) {
+        return StreamSupport.stream(((Iterable<SVKmer>)() ->
+                new SVKmerizerWithLowComplexityFilter(seq, kSize, minEntropy)).spliterator(), false);
+    }
+
+    public static Stream<SVKmer> stream( final byte[] seq, final int kSize, final double minEntropy ) {
+        return StreamSupport.stream(((Iterable<SVKmer>)() ->
+                new SVKmerizerWithLowComplexityFilter(seq, kSize, minEntropy)).spliterator(), false);
+    }
+
+    protected SVKmer nextKmer( SVKmer tmpKmer, int validBaseCount ) {
+        final double[] entropies = getEntropies(kSize);
```

Comment:
I find it pretty confusing to have a variable with the same name and type as a static variable on the class. Couldn't we just rename this method to `initializeEntropies` and then access entropies statically? Also this seems like it allows someone to do something bad by constructing two different SVKmerizers with different kSizes -- we might want to throw an exception if that happens, or alternatively allow for different 'entropies' arrays to be looked up statically.